src/auth/passwordreset/CSS/reset.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../commons/assets/variables/font.scss';
+@import '../../../commons/assets/variables/colors.scss';
+
+.container {
+  margin:20px auto;
+
+  h1 {
+    color: $primary-color;
+    font-family: $lato-font;
+    font-weight: 100;
+    font-size: 24px;
+  }
+
+  .form {
+    min-height: 500px;
+    width: 544px;
+    border: #cccccc solid 1px;
```

Comment:
@HabibSentongo consider referencing the color with the color variables instead of just declaring them directly.

Reply:
@alexxsanya this has been resolved, thank you.